RNWCPP/ReactUWP/Views/DatePickerViewManager.cpp

```diff
@@ -212,4 +212,9 @@ XamlView DatePickerViewManager::CreateViewCore(int64_t tag)
   return datePicker;
 }
 
+YGMeasureFunc DatePickerViewManager::GetYogaCustomMeasureFunc() const
+{
+  return DefaultYogaSelfMeasureFunc;
```

Comment:
Yes, verified. Works fine because yoga is still the one setting it and the property takes precedence over the self-measure results. Width & height vary independently here, so setting one means they other will still come via self-measure.